docs/docs/kafka-connect.md

```diff
@@ -77,6 +77,7 @@ for exactly-once semantics. This requires Kafka 2.5 or later.
 | iceberg.table.\<table name\>.partition-by  | Comma-separated list of partition fields to use when creating the table                                          |
 | iceberg.table.\<table name\>.route-regex   | The regex used to match a record's `routeField` to a table                                                       |
 | iceberg.control.topic                      | Name of the control topic, default is `control-iceberg`                                                          |
+| iceberg.control.group-id-prefix            | Prefix for the control consuler group, default is `cg-control`                                                   |
```

Comment:
There's a type here: I guess you mean `consumer` (not `consuler` :)).

I would rather name this property `iceberg.control.consumer-group-id-prefix` to keep the kafka semantic.